cli/src/main/resources/help/module.txt

```diff
@@ -107,3 +111,6 @@ ARGUMENTS
                    NOTE: this argument makes sense only when the module.xml file
                    is generated, i.e. when the --module-xml argument isn't
                    specified.
+
+--allow-non-existing-resources - Used in conjunction with --resources in order to create
```

Comment:
--allow-nonexistent-resources is better. "Nonexistent" is an ordinary English word.